cypress/src/e2e/profile/APIdelegering.js

```diff
@@ -185,13 +185,15 @@ describe('API delegering tests', () => {
 
     //click option in filter, click same option again and verify if it resets
     cy.contains('Testdepartement').click();
-    cy.contains('Testdepartement').siblings().should('be.checked');
+    cy.wait(200);
+    cy.get('input[type="checkbox"]').should('be.checked');
+
     cy.contains('Testdepartement').click();
     cy.contains('Testdepartement').siblings().should('not.be.checked');
 
     //select Testdepartement option from filter and verify only APIs with Testdepartement as provider is listed
     cy.contains('Testdepartement').click();
-    cy.contains('Testdepartement').siblings().should('be.checked');
+    cy.get('input[type="checkbox"]').should('be.checked');
```

Comment:
This is just a tip for the future, in case you need to verify that a specific checkbox is checked and not just any checkbox.
```tip
    cy.contains('button', `Testdepartement`).find('[type=checkbox]').should('be.checked');
```
By using find() we get the closest matching element of the given type (here: checkbox) within the context (here, the button containing the text Testdepartement), so that the checkbox we are asserting is the one that belong to Testdepartement instead of for example a different one belonging to Skatteetaten.